code/__HELPERS/lists.dm

```diff
@@ -265,6 +265,15 @@
 		i++
 	return null
 
+// Returns the first key to match the specified element. This is intended for lists which are injective functions.
+// Which is to say, two keys will not map to the same element.
+/proc/get_key_by_element(var/list/L, var/element)
+	for(var/key in L)
+		if(L[key] == element)
+			return key
+	return null
```

Comment:
I think returning null is the default behaviour of a proc.

Reply:
I think it's okay for clarity here, doesn't matter either way

Reply:
It's a trailing return you nerd

Reply:
In this situation it's standard since it's a helper